.github/workflows/aws_tfhe_fast_tests.yml

```diff
@@ -54,6 +54,7 @@ jobs:
         uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
         with:
           fetch-depth: 0
+          token: ${{ secrets.FHE_ACTIONS_TOKEN }}
```

Comment:
why is it required ?

Reply:
In the internal repo it is I think
@soonum do you confirm?

I can put this commit in another PR if needed

Reply:
I think its needed for the internal repo as its a private repo, tfhe-rs is a public repo so its not needed

Reply:
got it